### ISSUE TITLE
Latest version 2.0.13.32288 crashes both Excel and Access 2016 when opening VBE

### ISSUE THREAD

**Issue by Jaspos** [rubberduck-vba/Rubberduck#2878]
Just installed this yesterday. When set to load on start-up it always crashes Access when I try to open the VBE (the crash is fatal, I don't get into the VBE and then Access closes).

Yesterday it was fine with Excel, but now I am getting the same issue there too. FYI the splash screen for Rubberduck opens, and then the crash happens. 

When the crash happens I am given the option to close the prog or debug via Visual Studio (neither option keeps the program open).

No choice now but to uninstall as I cannot get into the VBE at all.

FYI I am using Win 10 640bit, but Office 2016 32-bit. Let me know if you need any more system/version info, or if there may be any log file of the error I can send to you

**Comment by Jaspos** [comment]
FYI not sure if it is relevant but I have two other VBE add-ins loaded (which work fine, but perhaps may interfere?)
- VBA Mz-Tools 3.0
- EverytingAccess.com vbWatchdog

**Comment by ThunderFrame** [comment]
Hi and thanks for raising an issue.

Are you able confirm a few things?

1. When you launch the VBE, are there any host-documents already open? Does it make a difference if there is a database file open when you launch VBE, or if Access doesn't have any databases open?

2. If there is a database open, does it have any forms/reports showing, or any code running before you launch the VBE?

3. With Rubberduck disabled, launch the VBE and close any open modules/userforms, then save. Re-enable Rubberduck and relaunch Access, load the same DB, and launch VBE. Does having all of the components in a closed state help with a stable launch?

4. Are you able to disable the other add-ins, relaunch Access, and see if the problem persists?

**Comment by Jaspos** [comment]
Hi there.

Happy to (try!) and help. Would love to get it back working for me again!

So to answer:

When you launch the VBE, are there any host-documents already open? Does it make a difference if there is a database file open when you launch VBE, or if Access doesn't have any databases open?
I have only tried this with a database open. But I did try it with a very simple test DB with no VBA code and the problem persisted. Also, same problem in Excel when opened with a blank workbook. I will try both with no open file and get back to you.

If there is a database open, does it have any forms/reports showing, or any code running before you launch the VBE
No, same problem both in no-code DBs, and DB opened with shift to ensure nothing was running/open

With Rubberduck disabled, launch the VBE and close any open modules/userforms, then save. Re-enable Rubberduck and relaunch Access, load the same DB, and launch VBE. Does having all of the components in a closed state help with a stable launch?
Currently can’t disable RubberDuck, as I can’t get in to the VBE to do so! I will uninstall, reinstall and re-try. Or is there a way to disable RubberDuck from outside the VBE without uninstalling and reinstalling?

Are you able to disable the other add-ins, relaunch Access, and see if the problem persists?
Again, I’ll try this and see what happens.

I’ll give these tings a try and get back to you (likely it will be tomorrow now). Would you prefer a response on this email, or via my issue post on GitHub?



Duncan

Duncan Salmon
www.alpine365.co.uk<http://www.alpine365.co.uk/> | facebook<https://www.facebook.com/pages/Alpine365/150986028258532?ref=hl> | twitter<https://twitter.com/alpine365ski>

[cid:image001.png@01D29E50.1C523E90]

*** Voted "One of the Best Alpine Chalets" by the Daily Telegraph ***
T: +33 (0)9 81 93 71 10 | M: +33 (0)6 66 03 53 14
Alpine365 SARL | 2 La Régence | 69 route d’Hauteville | 73700 Bourg St Maurice | France
No SIRET: 509 636 007 00015 | No TVA: FR47 509 636 007

**Comment by retailcoder** [comment]
@Jaspos Rubberduck's VBE add-in registry key has a value for *LoadBehavior* that the installer sets to 3 to load at startup - you can disable it without completely removing it by setting that value to 0:

![regedit](https://camo.githubusercontent.com/67cd1f7c2aa2466762dd698c7a1824ed488f2f1b/687474703a2f2f692e737461636b2e696d6775722e636f6d2f7a7a3853782e706e67)

**Comment by Jaspos** [comment]
Great, will try all this and let you know.



Duncan

Duncan Salmon
www.alpine365.co.uk<http://www.alpine365.co.uk/> | facebook<https://www.facebook.com/pages/Alpine365/150986028258532?ref=hl> | twitter<https://twitter.com/alpine365ski>

[cid:image001.png@01D29E54.4D1CF2F0]

*** Voted "One of the Best Alpine Chalets" by the Daily Telegraph ***
T: +33 (0)9 81 93 71 10 | M: +33 (0)6 66 03 53 14
Alpine365 SARL | 2 La Régence | 69 route d’Hauteville | 73700 Bourg St Maurice | France
No SIRET: 509 636 007 00015 | No TVA: FR47 509 636 007

**Comment by retailcoder** [comment]
I just noticed you have vbWatchdog installed... I need to check if the fix went into 2.0.13, but there is/was a parser issue with vbw-generated code.

**Comment by Jaspos** [comment]
Ok, I have been able to take a quick look now


1.     For both Excel and Access, when Rubberduck is set to load on start-up it crashes both XL & ACC when I launch the VBE, even with no workbook/DB open/no code running

2.     If not loaded at start-up, it still crashes both XL & ACC when in the VBE if I try to load RubberDuck

3.     If I disable vbWatchdog the problem persists

4.     If I disable both vbWatchdog and Mz-Tools (ie disable all other VBE add-ins I am running), the problem persists

So in summary, with no workbook or DBs open, no code running, and no other VBE add-ins loaded, Rubberduck still crashes ie I simply cannot get it to run on my machine.

FYI yesterday I did have it running for a while (it was fine in Excel, and if it didn’t load at start-up I could get into the VBE and then load it in Access and sometimes it would run fine).

But today, nada.

Sorry, this is probably not much help! But for now, for me, it is 100% a no-go.


Duncan

Duncan Salmon
www.alpine365.co.uk<http://www.alpine365.co.uk/> | facebook<https://www.facebook.com/pages/Alpine365/150986028258532?ref=hl> | twitter<https://twitter.com/alpine365ski>

[cid:image001.png@01D29E57.F785CC50]

*** Voted "One of the Best Alpine Chalets" by the Daily Telegraph ***
T: +33 (0)9 81 93 71 10 | M: +33 (0)6 66 03 53 14
Alpine365 SARL | 2 La Régence | 69 route d’Hauteville | 73700 Bourg St Maurice | France
No SIRET: 509 636 007 00015 | No TVA: FR47 509 636 007

**Comment by comintern** [comment]
Is this logging anything to the RD log file? Also when you say...

> the splash screen for Rubberduck opens, and then the crash happens

... do you mean that the splash screen is the only thing you see, or does it ever get as far as displaying the VBE window?

Finally - most of the time when you get debug prompt, that means the host process survived long enough to write an event to the Windows application log.  Can you check to see if there are any details there?  It would be in the Computer Management console under System Tools->Event Viewer->Windows Logs->Application.

**Comment by Jaspos** [comment]
OK….


1.     I only see the Rubbeduck splash screen, never get to see the VBE window. FYI I have attached a screenshot of the crash message in case it is of any use:



[cid:image002.png@01D29E62.3CDCC290]



2.     There ARE some Rubberduck logs, but they don’t appear related to the crashes, and only log my system info:

2017-03-15 19:17:25.0799;INFO-2.0.13.32288;Rubberduck.App;Rubberduck version 2.0.13.32288 loading:
            Operating System: Microsoft Windows NT 10.0.14393.0 x64
            Host Product: Microsoft Office 2016 x86
            Host Version: 16.0.7369.2118
            Host Executable: MSACCESS.EXE;

2017-03-16 10:28:08.0341;INFO-2.0.13.32288;Rubberduck.App;Rubberduck version 2.0.13.32288 loading:
            Operating System: Microsoft Windows NT 10.0.14393.0 x64
            Host Product: Microsoft Office 2016 x86
            Host Version: 16.0.7369.2118
            Host Executable: EXCEL.EXE;

FYI I have done a full search on my c drive for any files containing “rubberduck” and these are the only text files that show up



3.     Yes, I do have a windows fault log, I have attached the latest one here.

Hope some of this may help!



Duncan

**Comment by retailcoder** [comment]
Rubberduck must be logging something, however IIRC I believe logging is disabled by default (or did we change that? can't remember...) - anyway if you go to `%appdata%\Rubberduck`, there should be a `rubberduck.config` xml file in there.. if you could crash RD just once more for us, I think we can get something to chew on.

Open up the configuration file in Notepad (or Notepad++), and find the `<GeneralSettings>` section near the top:

````xml
    <GeneralSettings>
      <Language Code="en-US" />
      <ShowSplash>true</ShowSplash>
      <SmartIndenterPrompted>false</SmartIndenterPrompted>
      <AutoSaveEnabled>false</AutoSaveEnabled>
      <AutoSavePeriod>10</AutoSavePeriod>
      <MinimumLogLevel>0</MinimumLogLevel>
    </GeneralSettings>
````

Set the `MinimumLogLevel` to 0, save & close the file, then bring up the VBE with Rubberduck enabled - when it crashes, I expect to see a detailed exception & stack trace in the logs: if you can share the contents of that log file here, I'm sure we can find out exactly what's happening.

Thanks!

**Comment by retailcoder** [comment]
Oh crap, sorry I missed the comment about there having RD logs - ignore my last comment. :neutral_face:

**Comment by ThunderFrame** [comment]
I wonder if maybe the settings file is corrupt, and RD is choking on the file.

You could try deleting the settings file, and RD should recreate it.

The settings file is located at `%APPDATA%\Rubberduck\rubberduck.config`

**Comment by Jaspos** [comment]
OK, that worked. A bit. Here’s what happened:

Worked a few times – but each time I closed the main app I got an message as if the app had crashed (“Excel is trying to recover your data, yada yada yada”), and then it reopened the file. Same for Access & XL

But now it’s back as it was ie crashed before getting to the VBE.

So this may well be related to the config file as I have re-deleted and it does the same as above,

NB I did make a change to “which windows open when RD opens” and then I couldn’t get to the VBE again -  so I assume this will have altered the config file, so this could be the problem?



Duncan

Duncan Salmon
www.alpine365.co.uk<http://www.alpine365.co.uk/> | facebook<https://www.facebook.com/pages/Alpine365/150986028258532?ref=hl> | twitter<https://twitter.com/alpine365ski>

[cid:image001.png@01D29E96.776C8790]

*** Voted "One of the Best Alpine Chalets" by the Daily Telegraph ***
T: +33 (0)9 81 93 71 10 | M: +33 (0)6 66 03 53 14
Alpine365 SARL | 2 La Régence | 69 route d’Hauteville | 73700 Bourg St Maurice | France
No SIRET: 509 636 007 00015 | No TVA: FR47 509 636 007

**Comment by comintern** [comment]
> NB I did make a change to “which windows open when RD opens” and then I couldn’t get to the VBE again -  so I assume this will have altered the config file, so this could be the problem?

I'm about 90% sure that is related to the issue.  I'll see if I can replicate this with any settings combinations on my machine and do a bit of static analysis to try and identify bad code paths.

**Comment by Jaspos** [comment]
Cool.

If it helps the three windows I had selected were explorer, todo, and code analysis (? - can’t remember the exact name – the style cop one).

Don’t think I changed anything else.

I am based in Europe so likely will be off this now until tomorrow our time, will get back on to this as needed then.



Duncan

Duncan Salmon
www.alpine365.co.uk<http://www.alpine365.co.uk/> | facebook<https://www.facebook.com/pages/Alpine365/150986028258532?ref=hl> | twitter<https://twitter.com/alpine365ski>

[cid:image001.png@01D29E97.DB1E92F0]

*** Voted "One of the Best Alpine Chalets" by the Daily Telegraph ***
T: +33 (0)9 81 93 71 10 | M: +33 (0)6 66 03 53 14
Alpine365 SARL | 2 La Régence | 69 route d’Hauteville | 73700 Bourg St Maurice | France
No SIRET: 509 636 007 00015 | No TVA: FR47 509 636 007

**Comment by retailcoder** [comment]
We need to catch and log exceptions thrown on loading configuration...

**Comment by ThunderFrame** [comment]
I got a repro. My bad config file vs a new config file has more than just the window visibility differences, so I need to isolate the problem entries. There seems to be a mismatch in one of the inspection names for MalformedAnnotation/AnnotationMissingArgument, but I'm on my work PC now... Once I get back to my laptop, I'll dig into this more.

**Comment by Jaspos** [comment]
This made me think – would it be useful if I sent you a copy of my (presumably corrupt) config file after I have this problem?

**Comment by ThunderFrame** [comment]
@Jaspos It might be useful to see your config file, but I've just done some testing, and while toggling the visibility of Windows did recreate this crash, the crash seems to be related to the window hooks and not the config settings themselves.

While I was able to recreate the crash on startup by setting the visibility of Code Explorer and Code Inspections to `true`, I found that the crash *only* occurred when I had those windows arranged in the same docking "Frame" as the Properties window. When Rubberduck starts, it temporarily displays *all* of the ToolWindows, ensures they're sized appropriately (using `EnsureMinimumWindowSize`), and then closes them *unless* the startup visibility is set to True, in which case it is left visible.

I can have the window visibility set to `true` in the Rubberduck config, without the host crashing, but *only* if I have the ToolWindows docked in a different docking frame to the Properties window.

The screencap below shows Rubberduck crashing after a manual toolwindow movement (with the Properties window being in the same dock). I suspect this is the same crash that is being triggered by the display of ToolWindows in the startup process.

![toolwindowplacement](https://cloud.githubusercontent.com/assets/16574009/24448360/a326b302-14bf-11e7-9bf4-e7513b3fad29.gif)

**Comment by JoergBr** [comment]
Same issue here and also most probably related to the rearranged windows. I removed the config file and RubberDuck startet again. Here's the config file before I removed it:

<?xml version="1.0" encoding="utf-8"?>
<Configuration>
  <GeneralSettings>
    <Language Code="de-DE" />
    <ShowSplash>true</ShowSplash>
    <CheckVersion>true</CheckVersion>
    <SmartIndenterPrompted>false</SmartIndenterPrompted>
    <AutoSaveEnabled>false</AutoSaveEnabled>
    <AutoSavePeriod>10</AutoSavePeriod>
    <MinimumLogLevel>6</MinimumLogLevel>
  </GeneralSettings>
  <HotkeySettings>
    <Settings>
      <HotkeySetting>
        <Name>IndentProcedure</Name>
        <Key1>P</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>false</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>IndentModule</Name>
        <Key1>M</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>false</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>CodeExplorer</Name>
        <Key1>R</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>false</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>FindSymbol</Name>
        <Key1>T</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>false</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>InspectionResults</Name>
        <Key1>I</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>true</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>TestExplorer</Name>
        <Key1>T</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>true</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>RefactorMoveCloserToUsage</Name>
        <Key1>C</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>true</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>RefactorRename</Name>
        <Key1>R</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>true</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>RefactorExtractMethod</Name>
        <Key1>M</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>true</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>SourceControl</Name>
        <Key1>D6</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>true</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>RefactorEncapsulateField</Name>
        <Key1>E</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>true</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
      <HotkeySetting>
        <Name>ParseAll</Name>
        <Key1>`</Key1>
        <IsEnabled>true</IsEnabled>
        <HasShiftModifier>false</HasShiftModifier>
        <HasAltModifier>false</HasAltModifier>
        <HasCtrlModifier>true</HasCtrlModifier>
      </HotkeySetting>
    </Settings>
  </HotkeySettings>
  <ToDoListSettings>
    <ToDoMarkers>
      <ToDoMarker Text="NOTE " />
      <ToDoMarker Text="TODO " />
      <ToDoMarker Text="BUG " />
    </ToDoMarkers>
  </ToDoListSettings>
  <CodeInspectionSettings>
    <CodeInspections>
      <CodeInspection Name="ObjectVariableNotSetInspection" Severity="Suggestion" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="FunctionReturnValueNotUsedInspection" Severity="Warning" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="SelfAssignedDeclarationInspection" Severity="Hint" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="MoveFieldCloserToUsageInspection" Severity="Suggestion" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="EncapsulatePublicFieldInspection" Severity="Suggestion" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="EmptyStringLiteralInspection" Severity="Warning" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="ImplicitActiveSheetReferenceInspection" Severity="Warning" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="ImplicitActiveWorkbookReferenceInspection" Severity="Warning" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="MultipleFolderAnnotationsInspection" Severity="Error" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="ProcedureCanBeWrittenAsFunctionInspection" Severity="Suggestion" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="UseMeaningfulNameInspection" Severity="Suggestion" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="WriteOnlyPropertyInspection" Severity="Suggestion" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="UntypedFunctionUsageInspection" Severity="Hint" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="AssignedByValParameterInspection" Severity="Warning" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="ConstantNotUsedInspection" Severity="Warning" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="DefaultProjectNameInspection" Severity="Suggestion" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="ImplicitPublicMemberInspection" Severity="Hint" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="MultilineParameterInspection" Severity="Suggestion" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="NonReturningFunctionInspection" Severity="Error" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="ObsoleteCallStatementInspection" Severity="Suggestion" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="ObsoleteGlobalInspection" Severity="Suggestion" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="ObsoleteLetStatementInspection" Severity="Suggestion" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="ObsoleteTypeHintInspection" Severity="Suggestion" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="OptionBaseInspection" Severity="Hint" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="ParameterCanBeByValInspection" Severity="Suggestion" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="ParameterNotUsedInspection" Severity="Warning" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="ProcedureNotUsedInspection" Severity="Warning" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="UnassignedVariableUsageInspection" Severity="Error" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="VariableNotUsedInspection" Severity="Warning" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="VariableNotAssignedInspection" Severity="Warning" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="ImplicitByRefParameterInspection" Severity="Hint" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="ImplicitVariantReturnTypeInspection" Severity="Hint" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="MultipleDeclarationsInspection" Severity="Warning" InspectionType="MaintainabilityAndReadabilityIssues" />
      <CodeInspection Name="ObsoleteCommentSyntaxInspection" Severity="Suggestion" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="OptionExplicitInspection" Severity="Error" InspectionType="CodeQualityIssues" />
      <CodeInspection Name="VariableTypeNotDeclaredInspection" Severity="Warning" InspectionType="LanguageOpportunities" />
      <CodeInspection Name="MissingAnnotationArgumentInspection" Severity="Error" InspectionType="CodeQualityIssues" />
    </CodeInspections>
    <WhitelistedIdentifiers />
    <RunInspectionsOnSuccessfulParse>true</RunInspectionsOnSuccessfulParse>
  </CodeInspectionSettings>
  <UnitTestSettings>
    <BindingMode>LateBinding</BindingMode>
    <AssertMode>StrictAssert</AssertMode>
    <ModuleInit>true</ModuleInit>
    <MethodInit>true</MethodInit>
    <DefaultTestStubInNewModule>false</DefaultTestStubInNewModule>
  </UnitTestSettings>
  <IndenterSettings>
    <IndentEntireProcedureBody>true</IndentEntireProcedureBody>
    <IndentFirstCommentBlock>true</IndentFirstCommentBlock>
    <IndentEnumTypeAsProcedure>false</IndentEnumTypeAsProcedure>
    <IndentFirstDeclarationBlock>true</IndentFirstDeclarationBlock>
    <AlignCommentsWithCode>true</AlignCommentsWithCode>
    <AlignContinuations>true</AlignContinuations>
    <IgnoreOperatorsInContinuations>true</IgnoreOperatorsInContinuations>
    <IndentCase>false</IndentCase>
    <ForceDebugStatementsInColumn1>false</ForceDebugStatementsInColumn1>
    <ForceCompilerDirectivesInColumn1>false</ForceCompilerDirectivesInColumn1>
    <IndentCompilerDirectives>true</IndentCompilerDirectives>
    <AlignDims>false</AlignDims>
    <AlignDimColumn>15</AlignDimColumn>
    <EndOfLineCommentStyle>AlignInColumn</EndOfLineCommentStyle>
    <EndOfLineCommentColumnSpaceAlignment>50</EndOfLineCommentColumnSpaceAlignment>
    <IndentSpaces>4</IndentSpaces>
    <VerticallySpaceProcedures>true</VerticallySpaceProcedures>
    <LinesBetweenProcedures>1</LinesBetweenProcedures>
  </IndenterSettings>
  <WindowSettings>
    <CodeExplorerVisibleOnStartup>true</CodeExplorerVisibleOnStartup>
    <CodeInspectionsVisibleOnStartup>false</CodeInspectionsVisibleOnStartup>
    <SourceControlVisibleOnStartup>false</SourceControlVisibleOnStartup>
    <TestExplorerVisibleOnStartup>true</TestExplorerVisibleOnStartup>
    <TodoExplorerVisibleOnStartup>true</TodoExplorerVisibleOnStartup>
  </WindowSettings>
</Configuration>

**Comment by Jaspos** [comment]
FYI I haven’t had a chance to look at this again yet, but I can confirm that I definitely had docked windows, including Code Explorer docked in the same frame as the Properties window.




Duncan

Duncan Salmon
www.alpine365.co.uk<http://www.alpine365.co.uk/> | facebook<https://www.facebook.com/pages/Alpine365/150986028258532?ref=hl> | twitter<https://twitter.com/alpine365ski>

[cid:image001.png@01D2ABCF.0BCD7580]

*** Voted "One of the Best Alpine Chalets" by the Daily Telegraph ***
T: +33 (0)9 81 93 71 10 | M: +33 (0)6 66 03 53 14
Alpine365 SARL | 2 La Régence | 69 route d’Hauteville | 73700 Bourg St Maurice | France
No SIRET: 509 636 007 00015 | No TVA: FR47 509 636 007

**Comment by JoergBr** [comment]
Me too had docked windows on the left and on the bottom of the VBE window for RubberDuck.

**Comment by JoergBr** [comment]
I've been able to narrow things down a bit. After removing the config file and restarting RubberDuck, the crash reoccured when I tried to show the "ToDo Items" which I had docked to the Direct Window before. I removed the config file again, restarted RubberDuck and then opened the Options Menu of the VBE first. On the Docking tab, I disabled the docking of the Direct Window. After opening the "ToDo Items", RubberDuck did NOT crash. So it must be an issue related with the docking to the Direct Window which was at the window bottom.
